app/src/main/res/layout/view_unlock.xml

```diff
@@ -45,10 +44,85 @@
         android:gravity="center_horizontal"
         android:lines="2"
         android:maxLines="2"
-        android:text="@string/actions_enter_passcode_or_fingerprint" />
+        android:text="@string/actions_enter_passcode_or_fingerprint"
+        android:visibility="gone" />
 
     <com.saltedge.authenticator.widget.passcode.PasscodeInputView
         android:id="@+id/passcodeInputView"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        android:layout_marginTop="@dimen/dp_115" />
+
+    <LinearLayout
+        android:id="@+id/resetPasscodeLayout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <ImageView
+            android:id="@+id/backImageView"
+            android:layout_width="@dimen/dp_24"
+            android:layout_height="@dimen/dp_24"
+            android:layout_marginStart="@dimen/dp_16"
+            android:layout_marginTop="@dimen/dp_16"
+            android:scaleType="fitCenter"
+            android:src="@drawable/ic_back_24dp" />
+
+        <ImageView
+            android:id="@+id/imageView"
+            android:layout_width="@dimen/dp_328"
+            android:layout_height="@dimen/dp_200"
+            android:layout_gravity="center_horizontal"
+            android:layout_marginStart="@dimen/dp_16"
+            android:layout_marginTop="@dimen/dp_125"
+            android:layout_marginEnd="@dimen/dp_16"
+            android:contentDescription="@null"
+            android:scaleType="fitCenter"
+            android:src="@drawable/ic_onboarding_page_1"
+            tools:background="@color/primary" />
+
+        <TextView
+            android:id="@+id/titleView"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/dp_32"
+            android:layout_marginTop="@dimen/dp_30"
+            android:layout_marginEnd="@dimen/dp_32"
+            android:ellipsize="end"
+            android:fontFamily="@font/roboto_regular"
+            android:maxLines="2"
+            android:textAlignment="center"
+            android:textColor="@color/primary_text"
+            android:textSize="@dimen/text_27"
```

Comment:
wrong size. as i see in design, it is 30sp

Reply:
all texts has common style. please use or create them. This will prevent to do double work.